java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/PythonApiIT.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.systemtest.suite;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.io.TempDir;
+
+import sleeper.configuration.properties.table.TableProperty;
+import sleeper.systemtest.suite.dsl.SleeperSystemTest;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.stream.LongStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static sleeper.systemtest.suite.fixtures.SystemTestInstance.MAIN;
+import static sleeper.systemtest.suite.testutil.TestContextFactory.testContext;
+
+@Tag("SystemTest")
+public class PythonApiIT {
+    @TempDir
+    private Path tempDir;
+    private final SleeperSystemTest sleeper = SleeperSystemTest.getInstance();
+
+    @BeforeEach
+    void setup() {
+        sleeper.connectToInstance(MAIN);
+    }
+
+    @Nested
+    @DisplayName("Ingest files")
+    class IngestFiles {
+        @BeforeEach
+        void setup() {
+            sleeper.reporting().startRecording();
+        }
+
+        @AfterEach
+        void tearDown(TestInfo testInfo) {
+            sleeper.reporting().printIngestTasksAndJobs(testContext(testInfo));
+        }
+
+        @Test
+        void shouldBatchWriteOneFile() throws IOException, InterruptedException {
+            // Given
+            sleeper.localFiles(tempDir)
+                    .createWithNumberedRecords("file.parquet", LongStream.range(0, 100));
+
+            // When
+            sleeper.pythonApi(tempDir)
+                    .ingest().batchWrite("file.parquet")
```

Comment:
It's a bit difficult to tell here that this will take the files from the local directory, while the other tests won't.

Please can we give this a name that describes what it'll do in the terms used by the DSL (eg. ingest by queue uploading a local file)?

Can we also pass the temporary directory in directly here? It looks like it's not needed for the other methods on the Python API.